docs/pages/jpscore/jpscore_operativ.md

```diff
@@ -116,7 +116,7 @@ Usage:
   The repulsive force between two agents. See [Fig. 7](https://arxiv.org/pdf/1008.4297.pdf).
     - `nu` is the strength of the force ($$\nu$$ in Eq. (19)). 
     - `dist_max` is the maximum force at contact ($$f_m$$)
-    - `disteff_max`: cut-off radius ($$r_c$$)
+    - `disteff_max`: cut-off radius ($$r_c$$). Note this value should be smaller than `cell_size` of the linkedcells. See [Model parameters (in general)](#model-parameters-in-general).
```

Comment:
Can you also update the  `last_updated: Jul 22, 2020 ` part?

Reply:
done.